src/utils/user.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line import/prefer-default-export
+export const isRegularUser = (user) => {
+  if (!user) {
+    return false;
+  }
+
+  // todo: to change
+  return !user.get('email').includes('@graasp.org');
```

Comment:
Indeed this looks like it's bound to break. What's the actual logic going to be like?

Reply:
I'm not sure, currently light users are identical to regular users, except their automatically generated email as well as some extra data they have. Probably checking their additional data will be more accurate.